.github/workflows/release-to-candidate.yml

```diff
@@ -46,7 +46,7 @@ jobs:
   call-for-testing:
     name: 📣 Create call for testing
     needs: [release, get-architectures]
-    environment: "Candidate Branch"
+    environment: "LTS Branch"
     runs-on: ubuntu-latest
```

Comment:
Needs a channel argument in most of the steps, per: https://github.com/snapcrafters/sublime-merge/blob/3e350971a6a3394a10546d39963076b3ad9c3598/.github/workflows/release-to-candidate.yml#L46